.kitchen.yml

```diff
@@ -3,19 +3,15 @@ driver_plugin: vagrant
 driver_config:
   require_chef_omnibus: true
 
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: <%= ENV['BOOTSTRAP_VERSION'] || '11.16.4' %>
+
```

Comment:
This is site-specific.  We shouldn't drop such constraints in a community cookbook.